CONTRIBUTING.md

```diff
@@ -111,8 +111,9 @@ $ ./gradlew integrationTest
 #### Running integration tests on Android
 
 
-Running the integration tests on android requires the running of a dedicated test app,
-in addition to the requirements for Java SE integration tests. 
+Running the integration tests on android requires running the tests from an
+android context. For this there is a Test App in the `AndroidTest` directory.
+The test app picks up the source code from sync-core and sync-android respectively.
```

Comment:
Maybe clarify what this means, "The test application picks up source code from `sync-android` and `sync-core`. It therefore only runs tests from `sync-core` and `sync-android`."

"Respectively" doesn't make sense in this context.